Sources/PackageCollections/Storage/SQLitePackageCollectionsStorage.swift

```diff
@@ -126,28 +126,38 @@ final class SQLitePackageCollectionsStorage: PackageCollectionsStorage, Closable
     func put(collection: Model.Collection,
              callback: @escaping (Result<Model.Collection, Error>) -> Void) {
         DispatchQueue.sharedConcurrent.async {
-            do {
-                // write to db
-                let query = "INSERT OR REPLACE INTO \(Self.packageCollectionsTableName) VALUES (?, ?);"
-                try self.executeStatement(query) { statement -> Void in
-                    let data = try self.encoder.encode(collection)
-
-                    let bindings: [SQLite.SQLiteValue] = [
-                        .string(collection.identifier.databaseKey()),
-                        .blob(data),
-                    ]
-                    try statement.bind(bindings)
-                    try statement.step()
-                }
+            self.get(identifier: collection.identifier) { getResult in
```

Comment:
nit and not sure if its worth it, but maybe an "exists" API can be faster than a full get

Reply:
We do need the `Collection` to compare packages.

Reply:
missed that... 👍